server/flyway/sql/V28__create_silva_application.sql

```diff
@@ -0,0 +1,58 @@
+-- Create silva_dev, silva_test and silva_prod applications
+INSERT INTO app_fam.fam_application (
+    application_name,
+    application_description,
+    app_environment,
+    create_user,
+    create_date
+)
+VALUES ('SILVA_DEV', ' (DEV)', 'DEV', CURRENT_USER, CURRENT_DATE),
```

Comment:
Do you want to change their app to "RESULT" not "SILVA"?  I am just affraid silva will have more than one applications to be onboard to FAM in future and "**RESULT**" is the application name but "SILVA" is their team name.

Reply:
Silva is Team Silva's chosen name for 'New Results'

Reply:
Ah~! Sorry Conrad. I didn't know that. Thanks @gormless87 !